common/workunit/workunit.cpp

```diff
@@ -7297,6 +7297,34 @@ extern WORKUNIT_API StringBuffer &getClusterGroupName(StringBuffer &ret, const c
     return ret;
 }
 
+extern WORKUNIT_API ClusterType getClusterTypeByClusterName(const char *cluster)
+{
+    Owned<IEnvironmentFactory> factory = getEnvironmentFactory(true);
+    Owned<IConstEnvironment> env = factory->openEnvironment();
+    Owned<IPropertyTree> root = &env->getPTree();
+    StringBuffer path;
+    path.set("Software/ThorCluster[@name=\"").append(cluster).append("\"]");
+    IPropertyTree * child = root->queryPropTree(path);
+    if (child)
+    {
+        return ThorLCRCluster;
+    }
```

Comment:
The above approach is ok, but a bit verbose could be just:
```
if (root->hasProp(path))
    return ThorLCRCluster;
```

Reply:
I will fix it.